### DISCUSSION TITLE
B팀 1주차 주제 미션 PR올립니다.

### DISCUSSION THREAD

**Discussion by mindgitrwx** [lablup/contribution-academy-2021#7] [Show and tell]
@lizable 멘토님 PR 확인 및 머지 부탁드립니다!
PR: #5 

(1주차 설명에 나와있는 Reviewer에 멘토 추가의 경우, Repository 의 owner 가 아닌 경우 불가능해 보입니다.
https://stackoverflow.com/questions/32262295/adding-a-reviewer-to-a-github-pull-request)

**Comment by lizable** [comment]
@mindgitrwx [PR](https://github.com/lablup/contribution-academy-2021/pull/5#pullrequestreview-725291581)에 코멘트 남겼습니다!

**Comment by lizable** [comment]
@mindgitrwx merged!
